axelarnetwork/axelar-amplifier
# feat: integrate mdbook documentation tool

## Description

Integration with mdbook for high level documentation. This should be a documentation tool used in parallel with rustdoc to complement. Moved current README files to the book documentation and created a new top level README to point to the book. Currently many contracts documentation is empty, but we can start filling them up in future PRs

https://axelarnetwork.atlassian.net/browse/AXE-1925

## Todos

- Use anchors to reference code instead of hardcoding code snippets in documentation
- Fill in the documentation for the missing contracts

## Steps to Test
```bash
cargo install mdbook
cargo install mdbook-mermaid
cargo install mdbook-linkcheck
mdbook build
```

generated book should be in target/book/html

## Expected Behaviour

## Other Notes
